test/cql-pytest/test_tools.py

```diff
@@ -1180,7 +1180,10 @@ def _simple_table_with_keys(cql, keyspace: str, keys: Iterable[int]) -> tuple[st
     return table, schema
 
 
-def test_scylla_sstable_shard_of(cql, test_keyspace, scylla_path, scylla_data_dir) -> None:
+def test_scylla_sstable_shard_of(cql, test_keyspace, scylla_path, scylla_data_dir, has_tablets) -> None:
+    if has_tablets:
+        pytest.skip("tablets does not use fixed sstable-shard mapping")
```

Comment:
I think we should have a vnode and tablet variant of this test. Also I think we should have a fixture, which makes a certain test either tablet only or vnode only.

Reply:
but s `cql-pytest` test does not decide the settings of scylla instance under test, as the scylla instance is provided by `test.py` driver, unless the test creates another cluster / scylla instance using for instance the cluster manager.

would you recommend creating an additional scylla instance using the cluster manager for this scenario?

Reply:
also, could shed some lights on what would be the expect shard for the tablet variant? i am not sure how we can predict / determine which shard a certain token range should be owned?